cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -34,7 +34,7 @@ public class RegulatingControlMappingForStaticVarCompensators {
     }
 
     public static void initialize(StaticVarCompensatorAdder adder) {
-        adder.setRegulationMode(StaticVarCompensator.RegulationMode.OFF);
+        adder.setRegulating(false);
```

Comment:
```suggestion
        adder.setRegulating(false);
        adder.setRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE);
```
shouldn't we set the RegulationMode as `VOLTAGE` to avoid having a null RegulationMode in those kind of cases?@geofjamg